SpatialConnect/SCBackendService.m

```diff
@@ -369,8 +369,13 @@ - (RACSignal *)send:(SCSpatialFeature *)feature {
       SCDataStore *store = [dataService storeByIdentifier:[feature storeId]];
       id<SCSyncableStore> ss = (id<SCSyncableStore>)store;
       Msg *msg = [[Msg alloc] init];
-      msg.payload = [[ss generateSendPayload:feature] JSONString];
       msg.action = CREATE_FEATURE;
```

Comment:
I don't think that action is in the schema. https://github.com/boundlessgeo/schema/blob/master/actions.json

Should we create that or edit `CREATE_FEATURE`?